### PR TITLE
Compatibility

### PR DESCRIPTION
**Pull request by ElectricMaxxx**
I am targeting this branch, because {reason}.

Cause i want to use it in next version.
## Changelog
```Markdown
### Added
- version 2.0 for `phpcr-odm`
```
## Subject

I will add step by step changes to get symfony 4.0 running on cmf bundles and so here too, but i will do it step by step in separate PRs.

### CODE REVIEW

**Comment by greg0ire** [comment]
Please format your commit message according to our rules : 

- The commit message subject must be less than 50 characters and tell us *what* you did.
- The commit message body should tell us *why* you did it. It is optional but highly recommended.

Bad example :

```
Fixed bug #989 by removing call to defraculate()
```

Good example:

```
Remove call to defraculate()

Calling this function caused a bug because it interferes 
with calls to getSchmeckles().
Fixes #989
```

More details in CONTRIBUTING.md

**Comment by greg0ire** [comment]
Why did you target master?

**Comment by ElectricMaxxx** [comment]
Sorry, changed it.

File: composer.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     ],
     "require": {
         "php": "^7.1",
-        "doctrine/phpcr-bundle": "^1.1",
+        "doctrine/phpcr-bundle": "^1.1|^2.0",
```
Comment (core23) [inline composer.json]:
```diff
-        "doctrine/phpcr-bundle": "^1.1|^2.0",
+        "doctrine/phpcr-bundle": "^1.1 || ^2.0",
```

Comment (ElectricMaxxx) [reply composer.json]: That looks strange ...

Comment (ElectricMaxxx) [reply composer.json]: Is that the way to do it?

Comment (core23) [reply composer.json]: Just look at the following line ;)

**Comment by ElectricMaxxx** [comment]
@core23  better?

**Comment by greg0ire** [comment]
Hey there!

It looks like you did not fill the pull request template properly. Maybe you
were confused, so let me walk you through this step by step.

# Explaining why you target this branch

> I am targeting this branch, because {reason}.

Here, you are supposed to replace `{reason}` with something like:

- "there is a BC-break" if you picked the master branch
- "there is no BC-break" if you picked the stable branch

**Pull request merged**